docs/moderne-cli/README.md

```diff
@@ -252,12 +252,13 @@ Now that the repository is configured, it's time to migrate it to Spring Boot 3
     > Running recipe org.openrewrite.java.spring.boot3.VERSION_RECIPE_SPRING_BOOT
 
     > spring-projects/spring-petclinic@main
-        ✅ Fix results at file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/run/20230921095147-SsGwh/fix.patch
+        ✅ Fix results at file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/run/20230925102039-NVf33/fix.patch
+        ✅ Search results at file:///Users/mikesol/Desktop/code/spring-petclinic/.moderne/run/20230925102039-NVf33/search.patch
```

Comment:
I have no idea why we're producing search results for a migration recipe - but we are.